components/Buttons/src/MDCFloatingButton.h

```diff
@@ -20,24 +20,45 @@
 #import "MDCButton.h"
 
 /**
- Shapes for Material Floating buttons.
+ Types of Material Floating buttons.
```

Comment:
naming-nit: Considering existing discussion of shapes (samnm).  I would prefer style (Matches UIKit's ActivityIndicator) or type.  Also the shape is actually the same (circle), only the size is different.
Naming is hard.